packages/roosterjs-react-common/lib/plugins/PasteImagePlugin.ts

```diff
@@ -63,6 +63,7 @@ export default class PasteImagePlugin implements EditorPlugin {
         }
 
         // modify the pasting content and option so Paste plugin won't handle
+        placeholder.setAttribute("alt", image.name );
```

Comment:
this is replacing the attribute for the placeholder and not the final image, take a look at the image manager's replacePlaceholder

Reply:
also, image.name can be undefined, you'll have to allow for specifying a fallback alt value